### PR TITLE
Add statistics menu

### PR DESCRIPTION
**Pull request by djcouto**
Add statistics menu  …
* Refactor project

* Fix animations

* Change the Settings class to singleton and add contexts methods

* Create a AppsManager to manage applications installed

### CODE REVIEW

File: app/src/main/java/corp/kairos/adamastor/Home/HomeActivity.java [hunk 1]
```diff
@@ -43,48 +44,46 @@
...
-            public void onClick(View v) {
-                HomeApp ha = (HomeApp) v;
-                Intent i = pm.getLaunchIntentForPackage(ha.getPackageName());
-                HomeActivity.this.startActivity(i);
-            }
-        };
-    }
+            // Set animations
+            this.setDownActivity(AllAppsActivity.class);
+            this.setLeftActivity(StatisticsActivity.class);
```
Comment (hgg) [inline app/src/main/java/corp/kairos/adamastor/Home/HomeActivity.java]: And context list screen to the right :+1:

File: app/src/main/java/corp/kairos/adamastor/Settings/Settings.java [hunk 2]
```diff
@@ -72,104 +95,112 @@ public UserContext loadContextSettings(String context) {
...
             i = 0;
             for (UserContext context: contexts) {
                 if (context.appExists(app)) break;
                 else i++;
             }
-            if (i==3) zeroContext.addApp(app);
+            if (i==contextsNumber) zeroContext.addApp(app);
         }
 
+        System.out.println(zeroContext.toString());
```
Comment (hgg) [inline app/src/main/java/corp/kairos/adamastor/Settings/Settings.java]: Remove these prints please :)

**Review by hgg — CHANGES_REQUESTED** [review]
Please tend to the comments I've left you. Also, like discussed, change icons to their material design counterparts.

**Review by hgg — APPROVED** [review]

**Review by jlsilva1994 — APPROVED** [review]
Great work indeed. Everything working and you take your time refactoring all the code, i like it 👍

**Pull request merged**